Documentation/interacting_v3.md

```diff
@@ -180,4 +180,64 @@ compacted revision 5
 # any revisions before the compacted one are not accessible
 $ etcdctl get --rev=4 foo
 Error:  rpc error: code = 11 desc = etcdserver: storage: required revision has been compacted
+```
+
+## Grant leases
+
+Applicants can grant leases from an etcd cluster. Applications can also attach keys to leases. Each lease has a time-to-live (TTL) value. Applications specify the wanted TTL while granting the lease. The actual TTL value is determined by the etcd cluster and it will be at least as large as the one specified by the application. Once the timespan of TTL has elasped, the lease will be revoked. The key attached to the revoked lease are deleted.
+
+Here is the command to grant a lease:
+
+```
+# grant a lease with 10 second TTL
+$ etcdctl lease grant 10
+lease 32695410dcc0ca06 granted with TTL(10s)
+
+# attach key foo to lease 32695410dcc0ca06
+$ etcdctl put --lease=32695410dcc0ca06 foo bar
+OK
+```
+
+## Revoke leases
+
+Applicants can revoke a lease by specifing the lease ID. Keys attached to the revoked lease are deleted.
```

Comment:
Applications revoke leases by lease ID. Revoking a lease deletes all of its attached keys.